### ISSUE TITLE
Calendar: Initial view

### ISSUE THREAD

**Issue by VytautasLozickas** [primefaces/primereact#5157]
### Describe the feature you would like to see added

It would be nice if we could define the initial view of the calendar popup, for example, an `initialView="year"` option would show the year selector immediately after the popup opens and then allow the date selection to start from selecting the year (useful for selecting birthdate for example).

### Describe alternatives you have considered

Tried the existing `view` prop but it doesn't allow the selection to continue, i.e. setting `view="year"` closes the popup after selecting the year.

**Comment by melloware** [comment]
Can't you do this already?  https://primereact.org/calendar/#yearpicker

**Comment by VytautasLozickas** [comment]
Yes, I tried this already, but it doesn't quite work the way I'd like :)

> Describe alternatives you have considered
> Tried the existing view prop but it doesn't allow the selection to continue, i.e. setting view="year" closes the popup after selecting the year.

**Comment by melloware** [comment]
Understood I see what you are saying.
